security/webauthn/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>security-webauthn</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Quarkus QE TS: Security: WebAuth</name>
+    <dependencies>
```

Comment:
all tests you are adding needs a database and `quarkus.datasource.db-kind` is build-time property, therefore you know our QE Test FW will always re-build native executable and build performed my Quarkus Maven plugin will be ignored. You can spare few minutes by adding

```
<properties>
    <quarkus.build.skip>true</quarkus.build.skip>
</properties>
```